docs/runbooks/PromscaleIngestHighDataDuplication.md

```diff
@@ -0,0 +1,38 @@
+# PromscaleIngestHighDataDuplication
+
+## Meaning
+
+Client payload has either duplicates or retrying many a time for the
+data which has been already ingested.
+
+## Impact
+
+Ingestion performance will be poor
+
+## Diagnosis
+
+There will be a high volume of samples gets into Promscale when ingesting
+from HA Prometheus. It can be examined by running the following PromQL
+query in Grafana/Prometheus.
+
+```promql
+rate(promscale_ingest_duplicates_total{kind="sample"}[5m])
+```
+
+If more data points are seen as a result of the above query, follow
```

Comment:
I dont think this is right. It sounds like duplicates happen when using HA improperly. While that is a case, majority of duplicates are due to timeout for Prometheus remote-write, and not HA.

I will suggest to have something like this here
```
If you see high rate of duplicate samples, check Prometheus logs for timeout or batch retry errors. If found, refer to `Tune Prometheus remote-write config`.
```

Then, we need to decide on a call, what remote-write config should we suggest for mitigation. Lets keep this as standup agenda for next meeting.

After this, we should mention the edge case, which is the HA. This might be

```
Duplicate samples can also occur on a wrongly configured HA setup. If you are using Prometheus as HA, please refer to `Prometheus high availability` for mitigation.
```